src/dispatcher.js

```diff
@@ -213,11 +214,13 @@ async function makeDispatches(gitController) {
   }
 }
 
-async function getDispatchesFileContent(filePath, gitController) {
+async function getDispatchesFileContent(filePath, gitController, payloadCtx) {
   try {
-    return fs.readFileSync(filePath).toString('base64')
+    return await gitController.getFileContent(filePath, payloadCtx.ref)
```

Comment:
```suggestion
    return await gitController.getFileContent(filePath, payloadCtx.sha)
```
The link is built with the ctx.sha at https://github.com/prefapp/action-make-state-repos-dispatches/pull/124/files#diff-42bcdcfb5a8d0173682963987bbeec7c4002df45f9df607760e60528e4990249L209